### ISSUE TITLE
Tapping Information opens a webview within the app instead of opening video on Youtube App. 

### ISSUE THREAD

**Issue by horeyes** [sarbagyastha/youtube_player_flutter#299]
**Describe the bug**
Hello,

When I tap on the Avatar, title, or watch later the apps open youtube in a webview. I would like the app to launch youtube instead. Is there a method available that would allow me to do this? Ideally, I would've like to hide the avatar and information from the player, but it looks like it's require by the YoutubeIFrameApi. 

**To Reproduce**
When the player it's paused or started:
1. Tap Avatar or Title,
2. User Profile or video will open in a webview within the app in Desktop size.

**Expected behavior**
Open Youtube App

**Screenshots**
1. Video Info Displaying
![image](https://user-images.githubusercontent.com/12671900/87725406-42018980-c78b-11ea-9eba-18dd4b6d4f13.png)
2. Webview
![image](https://user-images.githubusercontent.com/12671900/87725261-f64ee000-c78a-11ea-9534-7f0475ee5422.png)

**Technical Details:**

**Code** 
import 'package:flutter/material.dart';
import 'package:youtube_player_iframe/youtube_player_iframe.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
        visualDensity: VisualDensity.adaptivePlatformDensity,
      ),
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key, this.title}) : super(key: key);

  final String title;

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    YoutubePlayerController _controller = YoutubePlayerController(
      initialVideoId: 'j4_Pj1rYpr8',
      params: YoutubePlayerParams(
        startAt: Duration(seconds: 0),
        showControls: true,
        showFullscreenButton: true,
        autoPlay: true,
        forceHD: true,
      ),
    );

    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Column(
        mainAxisSize: MainAxisSize.max,
        children: <Widget>[
          Container(
            height: MediaQuery.of(context).orientation == Orientation.portrait
                ? MediaQuery.of(context).size.height * .25
                : MediaQuery.of(context).size.height * .81,
            width: double.infinity,
            child: YoutubePlayerIFrame(
              controller: _controller,
              aspectRatio: 16 / 9,
            ),
          ),
        ],
      ),
    );
  }
}

**Comment by arodriguezgb** [comment]
I am having the same issue is there any workaround or settings we can change to fix this behaviour ?

**Comment by sarbagyastha** [comment]
@horeyes @arodriguezgb With **v1.0.1**, I have disabled navigation inside webview. So the issue of launching youtube inside webview shouldn't exist.

Also, added a method `hideTopMenu()` to hide those titles & share icons. But using it may violate YouTube's TOS.

**Comment by arodriguezgb** [comment]
Fantastic, thank you @sarbagyastha that was fast.

**Comment by HenglyEver** [comment]
I'm glad to hear that you have disabled navigation inside webview but please have a look at this, I think this considered as a bug, on hold of video player reveal the list of options (equivalent to right-click on youtube player), do you think that behavior should be disabled as well?
![youtube player](https://user-images.githubusercontent.com/35974032/87827224-a6643c00-c8a4-11ea-8dc4-577e999c96d3.jpg)

P.S. after upgrading to v1.0.1, the `autoPlay` param doesn't seem to work, can you please confirm? Thanks :D

**Comment by GauravPatni** [comment]
_controller.hideTopMenu() & _controller.hidePauseOverlay() notworking from me . 
Even after using above share, watch later, info etc are visible even for pause overlay.

Please check

**Comment by 97loser** [comment]
@HenglyEver  
 Have you got any solutions

> I'm glad to hear that you have disabled navigation inside webview but please have a look at this, I think this considered as a bug, on hold of video player reveal the list of options (equivalent to right-click on youtube player), do you think that behavior should be disabled as well?
> ![youtube player](https://user-images.githubusercontent.com/35974032/87827224-a6643c00-c8a4-11ea-8dc4-577e999c96d3.jpg)
> 
> P.S. after upgrading to v1.0.1, the `autoPlay` param doesn't seem to work, can you please confirm? Thanks :D

**Comment by Uvelir** [comment]
> I'm glad to hear that you have disabled navigation inside webview but please have a look at this, I think this considered as a bug, on hold of video player reveal the list of options (equivalent to right-click on youtube player), do you think that behavior should be disabled as well?
> ![youtube player](https://user-images.githubusercontent.com/35974032/87827224-a6643c00-c8a4-11ea-8dc4-577e999c96d3.jpg)
> 
> P.S. after upgrading to v1.0.1, the `autoPlay` param doesn't seem to work, can you please confirm? Thanks :D

The same happened to me on iOS, package version 7.0.0+7

**Comment by muhajirdev** [comment]
yeah same thing happend to m3 @Uvelir , did you found a solution?

**Comment by suyog-bst** [comment]
Regarding this feature, app redirection seems to be working fine in v2.1.0 but clicking the title or YouTube logo doesn't seem to work anymore.

Can this feature be restored please.
